sechub-pds-solutions/findsecuritybugs/env-cluster-initial

```diff
@@ -19,4 +19,5 @@ S3_ACCESSKEY=user
 S3_SECRETKEY=Why$5forThat?
```

Comment:
This file should be removed. For more information have a look at: https://github.com/mercedes-benz/sechub/pull/1862#issuecomment-1382224019.